src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vmss/nested_templates/load_balancer_new.json

```diff
@@ -0,0 +1,348 @@
+    "ipTemplateFilePaths": {
+      "existingId": "[variables('publicIpTemplateUriEmpty')]",
+      "existingName": "[variables('publicIpTemplateUriEmpty')]",
+      "new": "[variables('publicIpTemplateUri')]",
+      "none": "[variables('publicIpTemplateUriEmpty')]"
+    },
+    "lbId": "[resourceId('Microsoft.Network/loadBalancers',parameters('loadBalancerName'))]",
+    "publicIpAddressIds": {
+      "existingId": "[parameters('publicIpAddress')]",
+      "existingName": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.Network/publicIPAddresses/',parameters('publicIpAddress'))]",
+      "new": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.Network/publicIPAddresses/',parameters('publicIpAddress'))]",
+      "none": null
+    },
+    "subnetDeploymentName": "[concat('Subnet',parameters('loadBalancerName'))]",
+    "subnetIds": {
+      "existingId": "[parameters('subnet')]",
+      "existingName": "[concat(variables('vnetId'),'/subnets/',parameters('subnet'))]",
+      "new": "[concat(variables('vnetId'),'/subnets/',parameters('subnet'))]",
+      "none": null
+    },
+    "vnetTemplateUri": "https://azuresdkci.blob.core.windows.net/templatehost/CreateVnet_2016-07-18/azuredeploy.json",
+    "vnetTemplateUriEmpty": "https://azuresdkci.blob.core.windows.net/templatehost/CreateVnet_2016-07-18/azuredeploy_empty.json",
+    "subnetTemplateFilePaths": {
+      "existingId": "[variables('vnetTemplateUriEmpty')]",
+      "existingName": "[variables('vnetTemplateUriEmpty')]",
+      "new": "[variables('vnetTemplateUri')]",
+      "none": "[variables('vnetTemplateUriEmpty')]"
+    },
+    "vnetId": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.Network/virtualNetworks/',parameters('virtualNetworkName'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Network/loadBalancers",
+      "name": "[parameters('loadBalancerName')]",
+      "location": "[parameters('location')]",
+      "tags": "[parameters('tags')]",
+      "apiVersion": "2015-06-15",
+      "dependsOn": [
+        "[concat('Microsoft.Resources/deployments/', variables('ipDeploymentName'))]",
+        "[concat('Microsoft.Resources/deployments/', variables('subnetDeploymentName'))]"
+      ],
+      "properties": {
+        "backendAddressPools": [
+          {
+            "name": "[parameters('backendPoolName')]"
+          }
+        ],
+        "inboundNatPools": [
+          {
+            "name": "[parameters('natPoolName')]",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id": "[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPortRangeStart": "50000",
+              "frontendPortRangeEnd": "50119",
```

Comment:
Why do we give this static defaults instead of exposing options so the user can set them?